modules/nf-core/sequali/environment.yml

```
@@ -0,0 +1,7 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - "bioconda::sequali=0.12.0"
```

You do not need the " here.
```suggestion
  - bioconda::sequali=0.12.0
```